sky/global_user_state.py

```diff
@@ -294,6 +304,9 @@ def get_cluster_from_name(
     rows = _DB.cursor.execute('SELECT * FROM clusters WHERE name=(?)',
                               (cluster_name,)).fetchall()
     for row in rows:
+        # Explicitly specify the number of fields to unpack, so that
+        # we can add new fields to the database in the future without
+        # breaking the previous code.
```

Comment:
We should consider namedtuple in the future.

Reply:
Added the comment below. Thanks!

Reply:
Added the comment below. Thanks!